### PR TITLE
Docs link check

### PR DESCRIPTION
**Pull request by lochhh**
## Description

**What is this PR**

- [ ] Bug fix
- [ ] Addition of a new feature
- [x] Other

**Why is this PR needed?**
This PR 
- checks for broken/redirected links in the documentation using the Sphinx [linkcheck](https://www.sphinx-doc.org/en/master/usage/builders/index.html#sphinx.builders.linkcheck.CheckExternalLinksBuilder) builder.
- centralises websites commonly referenced in the documentation to ease the management of external links.

**What does this PR do?**
This PR
- updates redirected URLs to the redirect locations (as indicated by the Sphinx [linkcheck](https://www.sphinx-doc.org/en/master/usage/builders/index.html#sphinx.builders.linkcheck.CheckExternalLinksBuilder) builder).
- creates aliases to base URLs of websites referenced more than once by collecting these under `myst_url_schemes` in `conf.py`.

## References
This PR addresses #75. 

## How has this PR been tested?
The docs have been built locally. 

## Does this PR require an update to the documentation?
We may want to add linkcheck as an option in the docs: [building the docs locally](https://movement.neuroinformatics.dev/community/contributing.html#building-the-documentation-locally) and/or include this step in the [GitHub build docs workflow](https://github.com/neuroinformatics-unit/movement/blob/main/.github/workflows/docs_build_and_deploy.yml).

## Checklist:

- [x] The code has been tested locally
- [x] The documentation has been updated to reflect any changes
- [x] The code has been formatted with [pre-commit](https://pre-commit.com/)

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -132,3 +132,29 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 # html_static_path = ['_static']
+
+# The linkcheck builder will skip verifying that anchors exist when checking 
+# these URLs
+linkcheck_anchors_ignore_for_url = [
+    "https://gin.g-node.org/G-Node/Info/wiki/",
+    "https://neuroinformatics.zulipchat.com/",
+]
```
Comment (niksirbi) [inline docs/source/conf.py:141]: Out of my curiosity, why do these have to be ignored?

**Review by niksirbi — APPROVED** [review]
This is fantastic @lochhh! Will save us so much tume and headspace.

I have two comments:

1. Should we document this in the CONTRIBUTING guide? Perhaps mention in the [Editind the documetation](http://127.0.0.1:3000/docs/build/community/contributing.html#editing-the-documentation) section that external urls should be added to conf.py

2. I founc out I can run `sphinx-build -b linkcheck docs/source docs/build` locally to include link checking during the build. I have to add this to the [relevant GitHub action](https://github.com/neuroinformatics-unit/actions/tree/main/build_sphinx_docs), right?

File: docs/source/community/mission-scope.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 ## Mission
 
-[movement](movement-website:) aims to **facilitate the study of animal behaviour in neuroscience** by providing a suite of **Python tools to analyse body movements** across space and time.
+[movement](../index.md) aims to **facilitate the study of animal behaviour in neuroscience** by providing a suite of **Python tools to analyse body movements** across space and time.
```
Comment (niksirbi) [inline docs/source/community/mission-scope.md:6]: While we are at it, any chance we can replace this type of relative links with [explicit targets](https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html)? Should be more robust to files moving around.

Comment (lochhh) [reply docs/source/community/mission-scope.md]: Resolved in f890628

**Comment by niksirbi** [comment]
This all seems to work fine now. Should we merge?

**Comment by lochhh** [comment]
> This all seems to work fine now. Should we merge?

yep. Thanks for the review! I'll rebase #79 once this is merged.

**Pull request merged**